packages/flutter/test/material/outline_button_test.dart

```diff
@@ -43,121 +47,152 @@ void main() {
+              pathMatcher: coversSameAreaAs(clipPath,
+                  areaToCompare: clipRect.inflate(10.0)))
+          ..path(color: highlightedBorderColor, strokeWidth: borderWidth));
 
     // Tap gesture completes, button returns to its initial configuration.
     await gesture.up();
     await tester.pumpAndSettle();
     expect(
-      outlineButton,
-      paints
-        ..path(color: fillColor.withAlpha(0x00))
-        ..clipPath(pathMatcher: coversSameAreaAs(clipPath, areaToCompare: clipRect.inflate(10.0)))
-        ..path(color: borderColor, strokeWidth: borderWidth)
-    );
-    debugDisableShadows = true;
+        enabledOutlineButton,
+        paints
+          ..path(color: fillColor.withAlpha(0x00))
+          ..clipPath(
+              pathMatcher: coversSameAreaAs(clipPath,
+                  areaToCompare: clipRect.inflate(10.0)))
+          ..path(color: borderColor, strokeWidth: borderWidth));
   });
 
-
-  testWidgets('OutlineButton contributes semantics', (WidgetTester tester) async {
+  testWidgets('OutlineButton contributes semantics',
+      (WidgetTester tester) async {
     final SemanticsTester semantics = new SemanticsTester(tester);
     await tester.pumpWidget(
       new Directionality(
         textDirection: TextDirection.ltr,
         child: new Material(
           child: new Center(
-            child: new OutlineButton(
-              onPressed: () { },
-              child: const Text('ABC')
-            ),
+            child:
+                new OutlineButton(onPressed: () {}, child: const Text('ABC')),
           ),
         ),
       ),
     );
 
-    expect(semantics, hasSemantics(
-      new TestSemantics.root(
-        children: <TestSemantics>[
-          new TestSemantics.rootChild(
-            actions: <SemanticsAction>[
-              SemanticsAction.tap,
-            ],
-            label: 'ABC',
-            rect: new Rect.fromLTRB(0.0, 0.0, 88.0, 36.0),
-            transform: new Matrix4.translationValues(356.0, 282.0, 0.0),
-            flags: <SemanticsFlag>[
-              SemanticsFlag.isButton,
-              SemanticsFlag.hasEnabledState,
-              SemanticsFlag.isEnabled,
+    expect(
+        semantics,
+        hasSemantics(
+          new TestSemantics.root(
+            children: <TestSemantics>[
+              new TestSemantics.rootChild(
+                actions: <SemanticsAction>[
+                  SemanticsAction.tap,
+                ],
+                label: 'ABC',
+                rect: new Rect.fromLTRB(0.0, 0.0, 88.0, 36.0),
+                transform: new Matrix4.translationValues(356.0, 282.0, 0.0),
+                flags: <SemanticsFlag>[
+                  SemanticsFlag.isButton,
+                  SemanticsFlag.hasEnabledState,
+                  SemanticsFlag.isEnabled,
+                ],
+              )
             ],
-          )
-        ],
-      ),
-      ignoreId: true,
-    ));
+          ),
+          ignoreId: true,
+        ));
```

Comment:
why this change? it doesn't seem more readable this way.

maybe dartfmt got confused because there's no trailing commas?

Reply:
I've removed dartfmt